ironfish/src/rpc/routes/config/uploadConfig.ts

```diff
@@ -73,10 +73,36 @@ export function setUnknownConfigValue(
   config.set(sourceKey, value as any)
 }
 
+function stringToStringArray(value: string): string[] | null {
+  if (!value.startsWith('[') || !value.endsWith(']')) {
+    return null
+  }
+
+  // special-case empty array
+  if (value === '[]') return []
```

Comment:
If we cut out the bracket checking, and make this work purely off CSV (but don't support escaping for now), it can work as a normal set too.

```sh
ironfish config:set bootstrapNode test.bn1.ironfish.network
```

or

```sh
ironfish config:set bootstrapNode test.bn1.ironfish.network,test.bn2.ironfish.network
```

Reply:
Sounds good, I took out some of the special-casing and added an example to `ironfish config:set --help`